### ISSUE TITLE
Sourcing bash/powerline.sh shows u'common' as prompt

### ISSUE THREAD

**Issue by heaths** [powerline/powerline#2037]
I've installed powerline successfully on other Windows machines using WSL, though those were Ubuntu 16.04 and this is the first using 18.04. I've tried both `apt install powerline` and `pip3 install powerline-status`, but using the same config (shown below), my prompt simply shows (twice for each prompt, likely representing segments):

```
u'common'
u'common'
```

I followed the general installation instructions, such that `which powerline-daemon` and `which powerline-config` work. Relevant .bashrc shown below:

```bash
# powerline
if [-f ~/.local/lib/python3.6/site-packages/powerline/bindings/bash/powerline.sh]; then
  powerline-daemon -q
  POWERLINE_BASH_CONTINUATION=1
  POWERLINE_BASH_SELECT=1
  # source ~/.local/lib/python3.6/site-packages/powerline/bindings/bash/powerline.sh
fi
```

And my config is simply:
```js
{
    "ext": {
        "shell": {
            "theme": "default_leftonly"
        }
    },
    "segments": {
        "left": [
            {
                "function": "powerline.segments.common.env.cwd"
            },
            {
                "function": "powerline.segments.common.vcs.branch",
                "status_colors": true
            }
        ]
    }
}
```

I can't figure out what this is, and - though I've tried many different queries - searching for even a similar reference has been impossible.

**Comment by StopMotionCuber** [comment]
Dumb question, but you commented out the source command.
Was this part of your debugging mechanism and you simply forgot to change that when pasting here or is this part of your error?

**Comment by heaths** [comment]
Yes, sorry. If I run the same command during bash init or after bash is running, the same thing happens. I.e. it doesn't work and is consistently showing `u'common'` twice whether or not that is commented out.
